lib/src/library.dart

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/element/element.dart';
+import 'package:analyzer/src/dart/resolver/scope.dart';
+
+/// A high-level wrapper API with common functionality for [LibraryElement].
+class Library {
```

Comment:
The name bothers me a bit. Seems a bit too generic.

Maybe LibraryHelper? Wrapper?

Not a huge deal...just thinking...